recipes/optuna-dashboard/meta.yaml

```
@@ -0,0 +1,55 @@
+{% set name = "optuna-dashboard" %}
+{% set version = "0.4.0" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/optuna-dashboard-{{ version }}.tar.gz
+  sha256: 29efd2e1df7e240d64546220a5086980073e2af25b62b2180ad305b5977ab116
+
+build:
+  number: 0
+  entry_points:
+    - optuna-dashboard = optuna_dashboard.cli:main
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - bottle
+    - optuna >=2.7
+    - python
+    - typing-extensions  # [py<38]
```

@hadim Sorry, it is required to add `scikit-learn` here.
https://github.com/optuna/optuna-dashboard/blob/a6af1358bad764a0d60e493ac02a1ee822ed36a0/setup.cfg#L25-L33

```suggestion
    - bottle
    - optuna >=2.7
    - python
    - typing-extensions  # [py<38]
    - scikit-learn
```